docs/en/user_guides/visualization.md

```diff
@@ -76,4 +78,94 @@ we can also run the following command to view them in TensorBoard:
 tensorboard --logdir work_dirs/test_visual/20220810_115248/vis_data
 ```
 
-If you would like to know more visualization usage, you can refer to [visualization tutorial](https://mmengine.readthedocs.io/en/latest/advanced_tutorials/visualization.html) in mmengie.
+### Visualize a Single Data Sample
+
+If you want to visualize a single data sample, a pair of input image and its ground truth in the dataset are necessary.
+You could prepare them on your own or download examples below by following commands:
+
+<div align=center>
+<img src="https://user-images.githubusercontent.com/24582831/189833109-eddad58f-f777-4fc0-b98a-6bd429143b06.png" width="70%"/>
+</div>
+
+```shell
+wget https://user-images.githubusercontent.com/24582831/189833109-eddad58f-f777-4fc0-b98a-6bd429143b06.png --output-document aachen_000000_000019_leftImg8bit.png
+wget https://user-images.githubusercontent.com/24582831/189833143-15f60f8a-4d1e-4cbb-a6e7-5e2233869fac.png --output-document aachen_000000_000019_gtFine_labelTrainIds.png
+```
+
+Then you can find their local path and use the scrips below to visualize:
```

Comment:
```suggestion
`SegLocalVisualizer` is a derivative from `Visualizer` in MMEngine and works for balabala. ...., for more details about `Visualizer` please refer to this tutorial.
```

Reply:
I removed this comments in line 83 where `SegLocalVisualizer` is introduced.